internal/service/fw/model_atcfw_access_code.go

```diff
@@ -86,7 +86,6 @@ var AtcfwAccessCodeResourceSchemaAttributes = map[string]schema.Attribute{
 	},
 	"policy_ids": schema.ListAttribute{
 		ElementType:         types.Int64Type,
-		Optional:            true,
```

Comment:
If it is allowed in the API, and not here, can you write a detailed comment why we are doing it?

Reply:
written a comment